source/2020-05-20-global-accessibility-awareness-day.md

```diff
@@ -0,0 +1,52 @@
+5. Splattributes- Right now, `...attributes` do not have a guaranteed merge order. This is problematic in cases where some `aria-` attributes can have multiple values, and the order of those values matter. 
+
+The strike team meets every week on Wednesdays, and has made progress on proposed solutions. Keep an eye out for new RFCs in the next few weeks that especially cover adding support for unique page titles as well as missing default language declaration support. 
+
+Another thing that has come out of this strike team's work is that there are other things that we'd like to do for the long-term, and consideration for a more permanent accessibility working group is now being discussed. Want to participate in the strike team's work? Check out the #st-a11y channel on [Discord](https://discord.gg/emberjs), and let us know! The meetings are also open to anyone who wants to attend. 
+
+## Build More Accessible Ember Apps Today!
+
+So what can you do **today** to improve the accessibility of your Ember application? Here are a few suggestions:
+
+- Ask about accessibility at work. Is it already baked into your design and development workflow? This is a great day to get that conversation going if it isn't already!
+- Check your package.json file and see if the most recent version of ember-template-lint is there. Try running the linter inside of your project with the `yarn lint:hbs` command in the terminal. What issues does it show?
+- Install the [Accessibility Insights](https://accessibilityinsights.io/) extension in your browser and turn on the tab stops checker. Then check the tabbing order of your website. Does the focus move from left to right and top to bottom? This is an easy way to check for keyboard traps, too! This browser extension allows you to toggle different checks that will help you quickly test different accessibility concerns for your website, and give you additional guidance for any manual checks you might have to do.
+- Help out by writing new rules for [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint). Get stuck, or need a new issue to work on? There's a Discord channel for that too- check out #e-template-lint and get the help you need.
+
+Have you already done these things? How about browsing the internet today with a screen reader turned on, or only using your keyboard?
+
+## Closing
+
+By creating a better future for accessible application development, we create products that make it possible for *everyone* to use. My goal is to get accessibility into the design and development workflow so it is in the list of a product's design constraints. I believe Ember is the framework that will help you do just that. Avoid costly lawsuits and code rewrites by putting accessibility into the base requirements of your work, because when accessibility is considered first, everyone wins.
```

Comment:
```suggestion
I'd like to close by giving a shoutout to [CrowdStrike](https://www.crowdstrike.com/) and [LinkedIn](https://www.linkedin.com/), two companies that use Ember and are also actively working to make accessibility improvements to their products. CrowdStrike let us know that in May alone, they have improved the keyboard accessibility of some of their more complex graphs, enhanced their SVG graphs with custom roles, and improved focus outlines for radio buttons. Great job!! 

By creating a better future for accessible application development, we create products that make it possible for *everyone* to use. My goal is to get accessibility into the design and development workflow so it is in the list of a product's design constraints. I believe Ember is the framework that will help you do just that. Avoid costly lawsuits and code rewrites by putting accessibility into the base requirements of your work, because when accessibility is considered first, everyone wins.
```